library/src/morpheus-master/sass/modules/tool/sitestats/_sitestats.scss

```diff
@@ -215,469 +221,399 @@
     }
   }
 
-
   /** Summary Table */
   .summaryTable {
-    /*border: 1px dashed #cccccc;*/
     text-align: left;
-    /*margin-top: 20px;*/
     width: 100%;
   }
+
   .summaryTable .labelColumn {
     min-width: 150px;
-    /* IE Dynamic Expression to set the width, ignored by other browsers */
-    width: expression(document.body.clientWidth < 300 ? "150px" : "auto" );
   }
+
   label.indnt0 {
     font-weight: bold;
   }
+
   label.indnt1 {
     font-weight: normal;
   }
+
   .selector {
-    color: #555555;
+    color: var(--sakai-text-color-2);
     font-size: 85%;
     font-weight: normal;
     margin: 0pt;
   }
+
   .additionalInfo {
-    color: #555555;
+    color: var(--sakai-text-color-2);
     font-size: 85%;
     font-weight: normal;
     margin: 0pt;
     margin-top: 3px;
     float: right;
   }
+
   .description {
-    color: #333333;
+    color: var(--sakai-text-color-1);
     font-size: 100%;
     font-weight: normal;
     margin: 0pt;
   }
+
   .notes {
-    color: #333333;
+    color: var(--sakai-text-color-1);
     font-size: 90%;
     font-weight: normal;
     margin: 0pt;
   }
 
-
-  /** Ajax loading */
-  .ajaxLoading {
-    background: #ffffff url(/sitestats-tool/images/ajaxload.gif) no-repeat;
-    color: window;
-  }
-
-
   /** Tool/Event tree nodes in preferences (for Tomahawk tree2) */
   .nodeToolSelected,.nodeEventSelected  {
-    /*font-weight: bold;*/
     vertical-align:top;
   }
+
   .nodeToolUnselected,.nodeEventUnselected {
     vertical-align:top;
     font-weight: normal;
     font-style: italic;
     text-decoration: line-through;
-    color: #888888;
+    color: var(--sakai-text-color-2);
   }
+
   .nodeToolPartialSelected{
     vertical-align:top;
     font-weight: normal;
     font-style: italic;
-    color: #555555;
+    color: var(--sakai-text-color-2);
   }
+
   .eventTree ul {
     padding-left: 0;
     margin-left: 0;
   }
+
   .eventTree li {
     list-style: none;
   }
+
   .navIcon {
     cursor: pointer;
     float: left;
   }
-  .nodeIcon {
-    /*vertical-align: bottom;*/
-  }
+
   .childCount {
     display: none;
-    color: #555555;
+    color: var(--sakai-text-color-2);
     font-weight: normal;
   }
 
-
-  /** Wicket table sorting indicator */
-  .wicket_orderUp a span {
-    background-image: url('/library/image/sakai/sortascending.gif');
-    background-repeat: no-repeat;
-    background-position:right center;
-    padding-right: 16px;
-  }
-  .wicket_orderDown a span {
-    background-image: url('/library/image/sakai/sortdescending.gif');
-    background-repeat: no-repeat;
-    background-position: right center;
-    padding-right: 16px;
-  }
-
-
-  /** Simple charts */
-  .simpleChart {
-    position: relative;
-    display: inline-block;
-    height: 15px;
-    width: 100%;
-    border: 1px solid #777;
-    text-align: center;
-    vertical-align: middle;
-    font-family:Arial,Helvetica,sans-serif;
-    font-size: 11px;
-    font-weight: bold;
-    color: #000;
```

Comment:
Is this needed for the graphs?

Reply:
I don't think so, the charts seem to not use any of these classes from what i can tell.